services/QuillLMS/app/serializers/progress_reports/standards/standard_serializer.rb

```diff
@@ -15,10 +15,17 @@ class ProgressReports::Standards::StandardSerializer < ActiveModel::Serializer
              :timespent,
              :average_score,
              :standard_students_href,
-             :mastery_status
+             :mastery_status,
+             :is_evidence
+
+  # rubocop:disable Naming/PredicateName
+  def is_evidence
+    !!object.try(:is_evidence)
+  end
+  # rubocop:enable Naming/PredicateName
 
   def average_score
-    object.average_score.round(2)
+    object.average_score&.round(2) || 0
```

Comment:
Should we default this to `nil` instead of 0, or is it important that we have numerical values on the front-end?

Reply:
We expect the value to be numeric in places like ProficiencyEvaluator.